src/app/core/routing/routing.service.spec.ts

```diff
@@ -63,16 +64,16 @@ describe('RoutingService', () => {
     })
       .compileComponents()
       .then(() => {
-        routingService = TestBed.get(RoutingService);
-        store = TestBed.get(Store);
-        router = TestBed.get(Router);
+        routingService = TestBed.inject(RoutingService);
```

Comment:
TestBed.get has been deprecated in favor of typed TestBed.inject https://angular.io/api/core/testing/TestBed#get